lib/haml_lint/options.rb

```diff
@@ -32,7 +32,7 @@ def parse(args)
 
     private
 
-    def add_linter_options(parser) # rubocop:disable Metrics/MethodLength
+    def add_linter_options(parser) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
```

Comment:
Can roll this back and instead try to break this method up, but I figured if we're already disabling `MethodLength`, this would be appropriate to disable as well.